dotCMS/src/integration-test/java/com/dotcms/datagen/TestUserUtils.java

```diff
@@ -173,8 +173,10 @@ public static User getUser(final Role role, final String email,
             final String lastName, final String password)
             throws DotDataException {
         final List<User> users = APILocator.getUserAPI().getUsersByNameOrEmail(email, 0, 1);
-        if (UtilMethods.isSet(users)) {
-            return users.get(0);
+        if (UtilMethods.isSet(users) && !users.isEmpty()) {
+            User user = users.get(0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'user' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5288908)